torch/testing/_internal/common_methods_invocations.py

```diff
@@ -458,6 +458,36 @@ def sample_inputs_linalg_matrix_power(op_info, device, dtype, requires_grad):
 
     return inputs
 
+def sample_inputs_hsplit(op_info, device, dtype, requires_grad):
+    return (SampleInput(make_tensor((6,), device, dtype,
+                                    low=None, high=None,
+                                    requires_grad=requires_grad),
+                        args=(2, None),),
+            SampleInput(make_tensor((S, S, S), device, dtype,
+                                    low=None, high=None,
+                                    requires_grad=requires_grad),
+                        args=([1, 2, 3], None),),)
```

Comment:
I would remove the None from the `args` parameter, since the sentinel value isn't supposed to be passed, anyway (here and on the other SampleInputs, too)